sds_data_manager/lambda_code/SDSCode/database/models.py

```diff
@@ -100,4 +113,6 @@ class FileCatalog(Base):
     end_date = Column(DateTime, nullable=False)
     version = Column(String, nullable=False)
     extension = Column(String, nullable=False)
-    status_tracking_id = Column(Integer, ForeignKey("status_tracking.id"))
+    status_tracking_id = Column(
+        Integer, ForeignKey("status_tracking.id"), nullable=False
```

Comment:
I remember a conversation about wanting null to be valid here because a file wouldn't have a status id at the time it's entered into the catalogue. Did I misunderstand that?

Reply:
I believe so. The way I understood was that by the time we get to writing to catalog table, we would already have record in the status table and therefore, it shouldn't be null. That's my understanding at the moment, but it could change later if needed.